educe/stac/learning/features.py

```diff
@@ -1411,12 +1372,12 @@ def _fuse_corpus(corpus, postags):
     to_delete = []
     for key in corpus:
         if key.stage == 'unannotated':
-            corpus[key] = _fuse_edus(corpus[key], corpus[key], postags[key])
+            corpus[key] = fuse_edus(corpus[key], corpus[key], postags[key])
```

Comment:
Or more explicitly, your request that I comment the abuse made me wonder
what the motivation for the abusive usage was in the first place… so I was
thinking aloud, not trying to get out of commenting it or saying that the
approach was justified.

So just I know what I'm trying to comment.


On Thu, 28 May 2015 at 16:59 Eric Kow <eric@erickow.com> wrote:

> I'm not disputing that. I just didn't really remember what that was for.
> Yes fuse_edus does fusing and it does the type promotion stuff and here
> we're just using it for the latter
>
>
> On Thu, 28 May 2015 at 16:53 Mathieu Morey <notifications@github.com>
> wrote:
>
>> In educe/stac/learning/features.py
>> <https://github.com/irit-melodi/educe/pull/47#discussion_r31241363>:
>>
>> > @@ -1411,12 +1372,12 @@ def _fuse_corpus(corpus, postags):
>> >      to_delete = []
>> >      for key in corpus:
>> >          if key.stage == 'unannotated':
>> > -            corpus[key] = _fuse_edus(corpus[key], corpus[key], postags[key])
>> > +            corpus[key] = fuse_edus(corpus[key], corpus[key], postags[key])
>>
>> Still, the abuse remains.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/irit-melodi/educe/pull/47/files#r31241363>.
>>
>